nixnet/database/database.py

```diff
@@ -56,33 +57,89 @@ def __hash__(self):
     def __repr__(self):
         return '{}(handle={})'.format(type(self).__name__, self._handle)
 
-    def close(self, close_all_refs=False):
+    def close(self, close_all=True):
+        # type: (bool) -> None
+        """Closes the database object, or all database objects.
+
+        To simplify the task of closing all database objects you opened, you
+        can use the ``close_all`` parameter set to true (default); otherwise,
+        only the database object is closed.
+
```

Comment:
I noticed the LabVIEW docs group properties by object (e.g. - the python Cluster class groups properties similarly to a Cluster property node) and thought it might make sense to use LV docs. I now see, we should only use the C API docs.